src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml

```diff
@@ -81,17 +84,135 @@
+
+                        <!--This Grid contains controls for selecting the Language-->
+                        <Grid x:Name="GeneralTab" 
+                              Margin="12 0 0 0"
+                              HorizontalAlignment="Left">
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="*" />
+                                <RowDefinition Height="*" />
+                                <RowDefinition Height="*" />
+                                <RowDefinition Height="*" />
+                                <RowDefinition Height="*" />
+                            </Grid.RowDefinitions>
+                            <Grid Grid.Row="0">
+                                <StackPanel Orientation="Vertical">
+                                    <Label  Content="Language" 
+                                            Grid.Row="0"
+                                            Foreground="{StaticResource PreferencesWindowFontColor}"/>
+                                    <ComboBox Grid.Row="1" 
+                                              Width="213"
+                                              HorizontalAlignment="Left"
+                                              ItemsSource="{Binding Path=LanguagesList}"
+                                              SelectedItem="{Binding Path=SelectedLanguage}"
+                                              Style="{StaticResource NoBordersComboBoxStyle}">
+                                    </ComboBox>
+                                </StackPanel>
+                            </Grid>
+
+                            <!--This Grid contains controls for selecting Run Settings options-->
+                            <Grid Grid.Row="1">
+                                <Grid.RowDefinitions>
+                                    <RowDefinition Height="Auto" />
+                                    <RowDefinition Height="*" />
+                                </Grid.RowDefinitions>
+                                <Label Grid.Row="0"
+                                   Content="Default Run Settings" 
+                                   Foreground="{StaticResource PreferencesWindowFontColor}"/>
+                                <Grid Grid.Row="1">
+                                    <Grid.ColumnDefinitions>
+                                        <ColumnDefinition Width="Auto"/>
+                                        <ColumnDefinition Width="Auto"/>
+                                    </Grid.ColumnDefinitions>
+                                    <RadioButton Grid.Column="0" 
+                                                 MinWidth="80"
+                                                 Style="{StaticResource RunSettingsRadioButtons}"
+                                                 IsChecked="{Binding Path=RunSettingsIsChecked}"
+                                                 Content="Manual"/>
```

Comment:
Same here

Reply:
@QilongTang 
I found a similar entry in resource files, then I reused the existing one. see the changes in the commit: https://github.com/DynamoDS/Dynamo/pull/11542/commits/9035f3d2b7de82a15cb6098d2a56a00c1b06836d